### PR TITLE
Feature/contact form

### PR DESCRIPTION
**Pull request by 0xMALVEE**
added contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -291,5 +291,76 @@ <h3>Skills</h3>
 
   </section>
 
+  <section id="contact">
+    <div class="contact-main">
+        <div class="contact-heading">
+            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in </p>
+        </div>
+        <form class="contact-form" method="POST" action="#">
```
Comment (IngDavidTM) [inline index.html:299]: - [ ] While you did a good job creating the required form :+1:, as the project requires the form should use the `action` method that integrates the [Formspree service](https://formspree.io/html).

- Hint: Action value should be similar to `https://formspree.io/f/{form_id}`

File: index.html [hunk 2]
```diff
@@ -291,5 +291,76 @@ <h3>Skills</h3>
...
+                    <input type="text" name="name" id="user-name" placeholder="Full name" maxlength="30" required>
+                </li>
+                <li>
+                    <input type="email" name="email" id="email-address" placeholder="Email address" required>
+                </li>
+                <li>
+                    <textarea name="message" id="msg" maxlength="500" placeholder="Enter text here" required></textarea>
+                </li>
+                <li class="button-wraper">
+                    <button type="submit">Get In Touch</button>
```
Comment (IngDavidTM) [inline index.html:311]: - [ ] You did a fantastic job styling the form of your portfolio :+1:, but as the project requires the button interactions such as hover, active (when it's pressed), disabled, and enable (default state) should be styled as shown in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=MRKFHr4zFAXuzn3O-0).

Screenshot from 2023-02-21 04-27-06

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @team 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbayomiOlaoye — APPROVED** [review]
Hi Team 🥇, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Good UI implementation :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 
- Form submits correctly ✔️ 

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**